integration_tests/src/main/python/delta_lake_test.py

```diff
@@ -125,6 +125,25 @@ def create_delta(spark):
     assert_gpu_and_cpu_are_equal_collect(lambda spark: spark.read.format("delta").load(data_path),
                                          conf=confs)
 
+@allow_non_gpu('FileSourceScanExec')
+@delta_lake 
+def test_delta_read(spark_tmp_path):
```

Comment:
the name should be more specific. we are not testing just the read. It looks like the test case is 

``test_delta_log_query_with_deletion_vectors` because we disable the fallback for delta log and enable DV